script.js

```diff
@@ -37,3 +38,130 @@ open.addEventListener('click', () => {
     header.removeChild(window);
   });
 });
+
+// Project popup window
+const seeProject = document.querySelectorAll('.show-project');
+
+const projects = [
+  {
+    title: 'Tonic',
+    devs: ['CANOPY', '•', 'Back End Dev', '•', '2015'],
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent, Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.',
```

Comment:
- [ ] Kindly use the project array to populate the HTML file, Using an object iterator, and remove the static content in the html.
> eg
```

projects.forEach((project) => {
   .....
})
```

Reply:
Thanks for your review @aimemalaika 

I understand you mean, using `for each` instead of `const {title, devs, description, tags, image, liveButton, sourceButton, live, source, } = projects[index];`  I don’t understand how this affects the html code.

Reply:
- [ ] Hey @Farida-Hamid . What the previous reviewer meant for you to do was to add the entire work section dynamically, not just the Pop-ups, even the cards. That was one of the project requirements. Right now you have your Works section written as static HTML in the Index.html file, kindly make the works section using javascript. That way, when you add projects in the future you won't have to replicate code and just add it in the array of projects to display it in the works section.